Sources/VitalDevices/DeviceReading/GlucoseMeter1808.swift

```diff
@@ -103,21 +140,28 @@ class GlucoseMeter1808: GlucoseMeterReadable {
 }
 
 private func toGlucoseReading(data: Data?) -> QuantitySample? {
-  guard let data = data else {
+  guard let data = data, data.count >= 10 else {
```

Comment:
why `data.count >= 10`?

Reply:
All the mandatory fields of a glucose measurement record add up to 10 bytes